packages/ui/src/common/hooks/useNetwork.ts

```diff
@@ -12,7 +12,7 @@ export const useNetwork = () => {
   const networks = useMemo<NetworkType[]>(
     () => [
       'local',
-      'local-mocks',
+      ...(IS_IN_DEV_ENVIRONMENT ? ['local-mocks' as const] : []),
```

Comment:
So this should work:
```suggestion
      ...(IS_DEVELOPMENT ? ['local-mocks' as const] : []),
```